tests/multicomponent_compressible_averaging/solution/solution-00000.0000.gnuplot

```diff
@@ -0,0 +1,91 @@
+# This file was generated by the deal.II library.
+
+
+#
+# For a description of the GNUPLOT format see the GNUPLOT manual.
+#
+# <x> <y> <velocity> <velocity> <p> <T> <C_1> <density> <thermal_expansivity> <specific_heat> <viscosity> <thermal_conductivity> 
+6.371e+06 0 0 0 3.35849e+07 273 0.5 2400.19 2.39919e-05 1527.29 2.39999e+20 2.39999 
+6.15391e+06 1.64894e+06 0 0 8.74632e+07 273 0.5 2400.5 2.39788e-05 1527.27 2.39998e+20 2.39998 
```

Comment:
Hm. Interesting. In this test, all the volume-average properties look good, but the specific heat is different from what we expect.

Reply:
The text at the top of the prm only listed the expected isochoric specific heat, which is not returned by ASPECT. I've updated the text to include the predicted isobaric specific heat, which is 1527.3 J/K/kg, as output by the test.